templates/lobby.html

```diff
@@ -508,6 +515,21 @@
             applyRounds(parsed.data.round, maxRounds);
             applyPlayers(parsed.data.players);
 
+            drawer = "";
+            cached = parsed.data.players;
+            cached.forEach(function (player) {
+                if (player.state === 1 && player.id != ownID)
+                {
+                    drawer = player.name;
+                }
+            });
+
+            if (drawer != "")
+            {
+                document.getElementById('next-player').innerHTML = drawer;
+                turnDialog.style.visibility = "visible";
```

Comment:
Technically, this would need to be shown in "ready" as well, since you can hit F5 and the dialog would be gone.